ledgers/ccf/scripts/set_contract_enclave_check_attestation_flag.py

```diff
@@ -0,0 +1,115 @@
+                r.body, r.status_code))
+            sys.exit(-1)
+    except Exception as e:
+        LOG.error('failed to set contract enclave check-attestation flag: {}'.format(str(e)))
+        sys.exit(-1)
+
+# -----------------------------------------------------------------
+def Main() :
+    default_output = os.path.join(CCF_Keys, 'ledger_authority.pem')
+
+    parser = argparse.ArgumentParser(description='Fetch the ledger authority key from a CCF server')
+
+    parser.add_argument(
+        '--logfile',
+        help='Name of the log file, __screen__ for standard output',
+        default='__screen__',
+        type=str)
+    parser.add_argument(
+        '--loglevel',
+        help='Logging level',
+        default='WARNING',
+        type=str)
+
+
+    parser.add_argument('--interface', help='Host interface where CCF is listening', required=True)
+    parser.add_argument('--member-name', help="Name of the user being added", default = "memberccf", type=str)
+    parser.add_argument('--port', help='Port where CCF is listening', type=int, default=6600)
+
+    parser.add_argument('--check-attestation', default=False, help="enable attestation verification", action='store_true')
+    
+    options = parser.parse_args()
+    
+    # -----------------------------------------------------------------
+    LOG.remove()
+    if options.logfile == '__screen__' :
+        LOG.add(sys.stderr, level=options.loglevel)
+    else :
+        LOG.add(options.logfile)
+
+    # -----------------------------------------------------------------
+    network_cert = os.path.join(CCF_Keys, "networkcert.pem")
+    if not os.path.exists(network_cert) :
+        LOG.error('network certificate ({}) does not exist'.format(network_cert))
+        sys.exit(-1)
+
+    member_cert = os.path.join(CCF_Keys, "{}_cert.pem".format(options.member_name))
+    if not os.path.exists(member_cert) :
+        LOG.error('member certificate ({}) does not exist'.format(member_cert))
+        sys.exit(-1)
+
+    member_key = os.path.join(CCF_Keys, "{}_privk.pem".format(options.member_name))
+    if not os.path.exists(member_key) :
+        LOG.error('member key ({}) does not exist'.format(member_key))
+        sys.exit(-1)
+
+    try:
+        member_client = CCFClient(
+            options.interface,
+            options.port,
+            network_cert,
+            session_auth=Identity(member_key, member_cert, "member"),
+            signing_auth=Identity(member_key, member_cert, "member"),
+        )
+    except Exception as e:
+        LOG.error('failed to connect to CCF service : {}'.format(str(e)))
+        sys.exit(-1)
+
+    set_contract_enclave_check_attestation_flag(member_client, options)
+
+    LOG.info('successfully set contract enclave check-attestation flag ')
+    sys.exit(0)
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+Main()
```

Comment:
all of these would be a lot cleaner if we could assume that pdo client modules were installed. it would also make everything else a LOT more consistent and easy to maintain. we do not need these for ledger nodes, but they should be installable/runnable in the same way that every other pdo script is run on every other pdo node.